cmd/Makefile.am

```diff
@@ -7,6 +7,9 @@ dist_man_MANS =
 noinst_PROGRAMS =
 noinst_LIBRARIES =
 
+# FIXME: get this via something like pkgconf once it is defined there
+SYSTEMD_SYSTEM_ENV_GENERATOR_DIR=/usr/lib/systemd/system-environment-generators
```

Comment:
Not perfect but we might use:
```
pkg-config systemd --varbiale=systemdsystemgenaratordir
```

Reply:
When I run this (with the typo fixed) I get empty output.

Reply:
`systemdsystemgenaratordir` is a different thing. There was no var for system env generators dir, hence I did not bring it up.

Reply:
For reference:
```
maciek@galeon:~ pkg-config --print-variables systemd
containeruidbasemax
containeruidbasemin
dynamicuidmax
dynamicuidmin
systemgidmax
systemuidmax
catalogdir
modulesloaddir
binfmtdir
sysctldir
sysusersdir
tmpfilesdir
systemdshutdowndir
systemdsleepdir
systemdusergeneratordir
systemdsystemgeneratordir
systemduserunitpath
systemdsystemunitpath
systemduserconfdir
systemdsystemconfdir
systemduserpresetdir
systemduserunitdir
systemdsystempresetdir
systemdsystemunitdir
systemdutildir
prefix
pcfiledir

# not what we want
maciek@galeon:~ pkg-config --variable=systemdsystemgeneratordir systemd
/usr/lib/systemd/system-generators
```

Reply:
Indeed, let's leave it as-is.